### ISSUE TITLE
Quickstart-scala build error: silencer lib not found

### ISSUE THREAD

**Issue by richardkapolnai-da** [digital-asset/daml#9397]
Issuing these commands (Daml 1.11), a build error occurs:
```
daml new quickstart-scala --template quickstart-scala
cd quickstart-scala/
sbt "application/runMain com.daml.quickstart.iou.IouMain localhost 6865"
```

The output (suppressed a bit) :
```
$ sbt "application/runMain com.daml.quickstart.iou.IouMain localhost 6865"
[info] Loading settings for project quickstart-scala-build from plugins.sbt ...
[info] Loading project definition from /tmp/quickstart-scala/project
[info] Loading settings for project parent from build.sbt ...
daml.sdk.version = 1.11.1
[info] Set current project to quickstart-scala (in build file:/tmp/quickstart-scala/)
[info] Updating 
[info] Resolved  dependencies
[warn]  Note: Unresolved dependencies path:
[error] sbt.librarymanagement.ResolveException: Error downloading com.github.ghik:silencer-lib_2.12.8:1.7.1
[error]   Not found
[error]   not found: /home/richard/.ivy2/local/com.github.ghik/silencer-lib_2.12.8/1.7.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/com/github/ghik/silencer-lib_2.12.8/1.7.1/silencer-lib_2.12.8-1.7.1.pom
[error]         at lmcoursier.CoursierDependencyResolution.unresolvedWarningOrThrow(CoursierDependencyResolution.scala:247)
[error]         at lmcoursier.CoursierDependencyResolution.$anonfun$update$35(CoursierDependencyResolution.scala:216)
[error]         at ...
[error] (scala-codegen / update) sbt.librarymanagement.ResolveException: Error downloading com.github.ghik:silencer-lib_2.12.8:1.7.1
[error]   Not found
[error]   not found: /home/richard/.ivy2/local/com.github.ghik/silencer-lib_2.12.8/1.7.1/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/com/github/ghik/silencer-lib_2.12.8/1.7.1/silencer-lib_2.12.8-1.7.1.pom
```

**Comment by tamaskalcza-da** [comment]
Apparently that library does not exist: https://mvnrepository.com/artifact/com.github.ghik/silencer-lib.

**Comment by tamaskalcza-da** [comment]
Currently there is a workaround that issue by setting the Scala version in `build.sbt` to something like `2.12.11`, e.g.:
```
scalaVersion in ThisBuild := "2.12.11"
```

**Comment by cocreature** [comment]
We’ll fix it but note that the Scala bindings are deprecated.

**Comment by cocreature** [comment]
Fixed in #9401
